### PR TITLE
Switch to argparse for the graph_transitions command

### PR DESCRIPTION
**Pull request by blueyed**
This was added in Django 1.8, and optparse (the old way) was removed in 1.10.

This switch removes support for Django 1.7, which as of Django 1.9's
release is no longer a supported Django version.

Fixes https://github.com/kmmbvnr/django-fsm/issues/146.

### CODE REVIEW

**Comment by blueyed** [comment]
> RuntimeError: failed to execute ['dot', '-Tpng', '-O', '/tmp/tmpJ93GKj'], make sure the Graphviz executables are on your systems' path

Regarding tests: should be monkeypath `graphviz.render`, or install `dot` / `graphviz`?

File: tox.ini [hunk 1]
```diff
@@ -16,10 +15,6 @@ basepython =
 deps =
     py26: ipython==2.1.0
```
Comment (kiss90benedek) [inline tox.ini]: I think this line could be removed as well.

**Comment by kmmbvnr** [comment]
No needs to drop support for django 1.6/py26, just use `getattr(BaseCommand, 'option_list',[])`

**Comment by kmmbvnr** [comment]
>> Regarding tests: should be monkeypath graphviz.render, or install dot / graphviz?
Obviously none of these. The unit test should call management command without `outputfile` argument.

File: tests/testapp/tests/test_command.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+class TestCommand(TestCase):
+    def test_graph_transitions(self):
+        with NamedTemporaryFile(suffix='.png') as tmpfile:
+            call_command('graph_transitions', 'testapp.Application',
+                         outputfile=tmpfile.name)
+
+        # XXX: collects also all models from the tests, and breaks?!
+        # call_command('graph_transitions', 'testapp')
```
Comment (blueyed) [inline tests/testapp/tests/test_command.py]: Any idea about what is happening here?
Should it work with all the models from the tests, and/or can we make it behave the same way as when it is run from the testapp manually?

**Comment by kmmbvnr** [comment]
Travis broken, no activity from OP

**Comment by blueyed** [comment]
For reference: tt was adopted and merged in https://github.com/kmmbvnr/django-fsm/pull/155.

**Pull request closed without merging**